eva_submission/eload_utils.py

```diff
@@ -187,37 +188,77 @@ def download_file(url, dest):
     urllib.request.urlcleanup()
 
 
-def get_vep_cache_version_from_ensembl(assembly_accession):
+def get_vep_and_vep_cache_version(mongo_uri, db_name, assembly_accession):
+    vep_cache_version = get_vep_and_vep_cache_version_from_db(mongo_uri, db_name)
+    if not vep_cache_version:
+        vep_cache_version = get_vep_and_vep_cache_version_from_ensembl(assembly_accession)
+    return vep_cache_version
+
+
+def get_vep_and_vep_cache_version_from_db(mongo_uri, db_name):
+    logger.info(f"Getting vep_version and vep_cache_version from db: {db_name}")
+    vep_version_list = []
+    with pymongo.MongoClient(mongo_uri) as db:
+        cursor = db[db_name]['annotationMetadata_2_0'].find({})
```

Comment:
Instead of hardcoding the collection name here, would you be able to use the property that is being returned [here](https://github.com/EBIvariation/eva-submission/blob/ba18de2abf60ddd459c0f048258fc2576ef8e930/eva_submission/ingestion_templates.py#L90)?

Reply:
updated